src/seedu/addressbook/data/person/Person.java

```diff
@@ -11,15 +11,15 @@
 public class Person implements ReadOnlyPerson {
 
     private Name name;
-    private Phone phone;
+    private Contact phone;
```

Comment:
You should not change `Phone` object to `Contact`, this approach is totally wrong. Instead you should still declared it as `Phone`. You can check how one of your teammates did it, #1195.